src/pykeen/models/inductive/base.py

```diff
@@ -97,8 +100,9 @@ def _get_entity_representations_from_inductive_mode(
             raise ValueError(
                 f"{self.__class__.__name__} does not support the transductive setting (i.e., when mode is None)"
             )
-        if mode in self._mode_to_representations:
-            return self._mode_to_representations[mode]
+        key = f"{mode}_factory"
```

Comment:
why is this mode_factory and not just mode?

Reply:
For a `(key, value)` pair `torch.nn.ModuleDict` registers `value` as an *attribute* with name `key` (cf. [here](https://pytorch.org/docs/stable/_modules/torch/nn/modules/container.html#ModuleDict)). Since `ModuleDict` is a `Module`, too, it already has an attribute `training`, which is why we cannot have the key `training` in the dict. Thus, I suffixed them with `_factory`. I did not want to change the `InductiveMode` constants though, thus we need to derive a key different from the `mode`.

Reply:
okay just making sure you thought about this. feel free to merge when ready